sdk/cosmos/azure-cosmos/src/test/java/com/azure/cosmos/CosmosDiagnosticsTest.java

```diff
@@ -299,6 +303,15 @@ private void validateTransportRequestTimelineDirect(String diagnostics) {
         assertThat(diagnostics).contains("\"eventName\":\"completed\"");
     }
 
+    private void validateJson(String jsonInString) {
+        try {
+            ObjectMapper mapper = new ObjectMapper();
```

Comment:
this is test and it doesn't matter much. 
but in general you should not create a ObjectMapper per method invocation, This is costly timewise.

Reply:
Yes good catch, although its test, but it should be on class level , i will change it in next iteration